electrum/lnworker.py

```diff
@@ -636,6 +637,8 @@ def get_payment_value(self, info: Optional['PaymentInfo'], plist):
         return amount_msat, fee_msat, timestamp
 
     def get_lightning_history(self):
+        if not self.network:
+            return {}
```

Comment:
ah, thanks for noticing that --offline does not work with LN
It would be good to still show LN history items when offline though. Let me try to fix that -- it's mostly independent of this PR anyway.

Reply:
should work with https://github.com/spesmilo/electrum/commit/1d187d36f0a90b13a88c8bd2f8e4f89f518b451d

Reply:
thanks for the work :+1: , that's really great! rebased